src/CabanaPD_Particles.hpp

```diff
@@ -780,6 +781,54 @@ class Particles<MemorySpace, PMB, TemperatureDependent, Dimension>
 #endif
 };
 
+template <typename MemorySpace, typename ModelType, typename ExecSpace>
+auto createParticles( ExecSpace exec_space, CabanaPD::Inputs inputs )
+{
+    std::array<double, 3> low_corner = inputs["low_corner"];
```

Comment:
Lines 787-794 and not in the createParticles function in Lines 803 and 817. How does this work?

Reply:
Those functions accept all these values as separate inputs instead of `CabanaPD::Inputs inputs`

Reply:
What about the computation of m, which is not an input?

    int m = std::floor( delta /
                        ( ( high_corner[0] - low_corner[0] ) / num_cells[0] ) );
    int halo_width = m + 1; // Just to be safe.

Reply:
Never mind. I found we compute that in the example itself.